todo-app/src/TodoDesc.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import './todo.css'
+
+export default class TodoDesc extends React.Component {
+    render() {
+        return (
+            <dev className="toDo-Description" >* {this.props.todoDesc}</dev>
```

Comment:
Instead of using `*`, you could have used a `ul` element for the list and each todo item could be `li` element.